db/models/package.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Package = db.define('package', {
```

Comment:
Package needs a `name` field. We can allow null for custom-created packages, or just (later on) include a setter method that generates a package name from the selected base and add-on(s). By the same logic, we should allow the package `description` to be `null` OR set `defaultValue: ''` for the package `description`.